scene/gui/control.cpp

```diff
@@ -2941,6 +2941,9 @@ void Control::_bind_methods() {
 	ClassDB::bind_method(D_METHOD("set_mouse_filter", "filter"), &Control::set_mouse_filter);
 	ClassDB::bind_method(D_METHOD("get_mouse_filter"), &Control::get_mouse_filter);
 
+	ClassDB::bind_method(D_METHOD("set_pass_on_modal_close_click", "enabled"), &Control::set_pass_on_modal_close_click);
+	ClassDB::bind_method(D_METHOD("get_pass_on_modal_close_click"), &Control::pass_on_modal_close_click);
```

Comment:
Just noticed this, `pass_on_modal_close_click` should be renamed to `get_pass_on_modal_close_click` to be consistent with other getters.